docs/release-notes/RELEASE-NOTES-3.0.0.md

```diff
@@ -0,0 +1,111 @@
+# About this release
+
+[This][this-release] is the stable Fortuna release.
+It:
+* Adds all tokens migrated in phase 2 of the token migration. This takes effect at the Fortuna mainnet hardfork height.
+* Disables usage of provided StateChannels WebSocket API for generalized
+  accounts. This is done because FSM is not fully aware of generalized
+  accounts. It is highly recommended for state channels' users **not to
+  upgrade their accounts to generalized ones** as it might impact their state
+  channels' user experience.
+* Sets the Fortuna mainnet hard fork height to block 90800 (approximately Wed 5th June 11am CEST).
+* Changes the default database persistency behaviour (user configuration key `chain > persist`), persisting it by default.
+
+[this-release]: https://github.com/aeternity/aeternity/releases/tag/v3.0.0
+
+This release is backward compatible with `v3.0.0-rc.1` except for the protocol in the state channels WebSocket Noise endpoints.
```

Comment:
@velzevur @uwiger Could you please check this.

Reply:
```suggestion
This release is backward compatible with `v3.0.0-rc.1` except for the protocol in the state channels WebSocket endpoints.
```